DepositeCalcTests/Tests/LoginPageTests.cs

```diff
@@ -36,7 +36,62 @@ public void ValidLoginTest()
             calculatorPage.WeitForReady();
 
             // Assert
-            Assert.True(calculatorPage.IsOpened());
+            Assert.True(calculatorPage.IsOpened(), "Incorrect credentials");
+        }
+
+        [TestCase("test.@test.com")]
+        [TestCase(".test@test.com")]
+        [TestCase("@test.com")]
+        [TestCase("(),:;<>[]@test.com")]
+        [TestCase(" ")]
+        [TestCase("")]
+        [TestCase("@test.com")]
+        [TestCase("test@test@test.com")]
+        [TestCase("TestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTest@test.com")]
+
+        public void InvalidEmailRemindPasswordTests(string userEmail)
+        {
+            // Act
+            loginPage.RemindPasswordForm.Open();
+            var sendRemindResult = loginPage.RemindPasswordForm.RemindPassword(userEmail);
+
+            // Assert
+            Assert.IsFalse(sendRemindResult.IsSuccessful, "Operation was not successful");
```

Comment:
You've missed message check.
```suggestion
            // Assert
            Assert.IsFalse(sendRemindResult.IsSuccessful, "Operation was not successful");
            Assert.AreEqual(sendRemindResult.Message, "<error message>");
```